src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+function createMessage(message, messageClass) {
+  let newMess = message;
+
+  if (messageClass === 'error') {
+    newMess = 'First promise was rejected';
+  };
+
+  document.body.insertAdjacentHTML('beforeend', `
+    <div data-qa="notification" class="${messageClass}">
+      ${newMess}
+    </div>
+  `);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.buttons === 1) {
+      resolve('First promise was resolved');
+    };
+  });
+
+  setTimeout(() => reject(Error('First promise was rejected')), 3000);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', () => {
+    resolve('Second promise was resolved');
+  });
+});
```

Comment:
this promise resolves even on clicking additional mouse buttons(wheel). Make it work only for left and right clicks

Reply:
I guess done